src/test/datascience/notebook/remoteNotebookEditor.vscode.common.ts

```diff
@@ -186,14 +186,16 @@ export function sharedRemoteNotebookEditorTests(
             'Should have at least one remote controller'
         );
 
-        await createEmptyPythonNotebook(disposables);
+        // Don't wait for the kernel since we will select our own
+        await createEmptyPythonNotebook(disposables, undefined, true);
```

Comment:
This is one of the few test cases where we now don't select a kernel automatially. In remote we don't have a python interpreter set. This test manually picks a kernel so for the initial createEmpty don't have it wait for Kernel auto selection as it will never happen.